src/electron/main.ts

```diff
@@ -59,6 +62,42 @@ function onToggleDevTools(sender: any, show: boolean) {
     }
 }
 
+/**
+ * Adds standard cut/copy/paste/etc context menu comments when right clicking input elements
+ * @param browserWindow The browser window to apply the context-menu items
+ */
+function registerContextMenu(browserWindow: BrowserWindow): void {
```

Comment:
I don't have a good way to test this... any ideas?

Reply:
No idea sorry

Reply:
I don't know if snapshot testing would even show anything here so I'm not sure either.